coalib/misc/Constants.py

```diff
@@ -13,6 +13,11 @@
                  "happen. When asked for, the following information "
                  "may help investigating:")
 
+VERSION_CONFLICT_MESSAGE = ("There is a conflict in the version of a package "
+                            "you have installed and the requirements of "
+                            "coala. The following information may help "
+                            "to resolve this conflict: ")
```

Comment:
I think the user can just google it.
This is an open bug in pip - https://github.com/pypa/pip/issues/988
ideally the 2 packages that cause the version Conflict should never be installed.